examples/css/index.html

```diff
@@ -83,6 +83,30 @@ <h3 class="mistica-text-title2">Tag</h3>
                     <div class="mistica-tag-error">Error</div>
                 </div>
 
+                <h3 class="mistica-text-title2">TextFields</h3>
+                <h4 class="mistica-text-title1">TextField</h4>
+                <div class="vertical-group">
+                    <div class="mistica-text-field">
+                        <input id="text-field-1" type="text" placeholder=" " />
+                        <label for="text-field-1">Label</label>
+                    </div>
+                    <div class="mistica-text-field">
+                        <input id="text-field-1" type="text" placeholder="Some placeholder" />
+                        <label for="text-field-1">Label</label>
```

Comment:
Long labels is bugged. Can we limit to 1 line? (I know this limitation is not good for accessibility but another solution is increase input field height to fit all the label. This is something that we need to improve in react too, a pending task.

![image](https://github.com/user-attachments/assets/4af37836-8f87-43f4-9e73-b6ab67ec4018)

Reply:
should be fixed

Reply:
I don't think so 😬
![image](https://github.com/user-attachments/assets/52ba0b50-f84c-4e93-9d25-e39cedccf67a)
https://mistica-m2fh9sls4-flows-projects-65bb050e.vercel.app/mistica-css

Reply:
hmmm... this is how I see it 
![image](https://github.com/user-attachments/assets/6a6ece32-a37f-41cb-b087-ed140ef1d6ab)
which browser are you using?

Reply:
I've reproduced it in Firefox. I've made some changes to fix it. Please check again